### PR TITLE
pkg add: fcitx5-qt

### PR DESCRIPTION
**Pull request by eatradish**
https://gitlab.com/fcitx/fcitx5-qt

### CODE REVIEW

File: extra-i18n/fcitx5-qt/autbuild/defines [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PKGNAME=fcitx5-qt
```
Comment (RedL0tus) [inline extra-i18n/fcitx5-qt/autbuild/defines]:
```suggestion
PKGNAME=fcitx5-qt
PKGSEC=utils
```

Just like fcitx-qt5

**Comment by liushuyu** [comment]
Your folder name is incorrect, it shows `autbuild`

**Review by RedL0tus — CHANGES_REQUESTED** [review]

**Review by liushuyu — CHANGES_REQUESTED** [review]
Request changes

File: extra-i18n/fcitx5-qt/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,5 @@
+PKGNAME=fcitx5-qt
+PKGSEC=utils
+PKGDES="Qt library and IM module for fcitx5"
+PKGDEP="fcitx5"
+BUILDDEP="extra-cmake-modules git qt-4 qt-5"
```
Comment (liushuyu) [inline extra-i18n/fcitx5-qt/autobuild/defines:5]:
```suggestion
BUILDDEP="extra-cmake-modules git qt-4 qt-5"

CMAKE_AFTER='-DENABLE_QT4=OFF -DENABLE_QT5=ON'
```

Comment (liushuyu) [reply extra-i18n/fcitx5-qt/autobuild/defines:5]: And add the following to `autobuild/beyond`:

```bash
mkdir -p build_qt4
cd build_qt4
cmake .. ${CMAKE_DEF} ${CMAKE_AFTER} -DQT_QMAKE_EXECUTABLE=/usr/lib/qt4/bin/qmake -DENABLE_QT5=OFF -DENABLE_QT4=ON
make -j$(nproc)
make install DESTDIR="${PKGDIR}"

cd ..
```

**Review by liushuyu — CHANGES_REQUESTED** [review]
Apply these suggestions to build Qt 4/5 in one build attempt.

File: extra-i18n/fcitx5-qt/autobuild/defines (hunk 2 above)
Comment (liushuyu) [reply extra-i18n/fcitx5-qt/autobuild/defines:5]: Unresolved.

Didn't apply the second resolution.

Comment (liushuyu) [reply extra-i18n/fcitx5-qt/autobuild/defines:5]: **Incorrect!**

I think you misunderstood. 

Please add the resolution of `autobuild/beyond`, but not to apply the same suggestion twice!

**Review by liushuyu — CHANGES_REQUESTED** [review]
Incorrect resolution. Try again.

File: extra-i18n/fcitx5-qt/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+CMAKE_AFTER='-DENABLE_QT4=OFF -DENABLE_QT5=ON'
+
+CMAKE_AFTER='-DENABLE_QT4=OFF -DENABLE_QT5=ON'
+
+CMAKE_AFTER='-DENABLE_QT4=OFF -DENABLE_QT5=ON'
```
Comment (liushuyu) [inline extra-i18n/fcitx5-qt/autobuild/defines]: Now, remove the redundant statements.

**Review by liushuyu — CHANGES_REQUESTED** [review]
Clean up

File: extra-i18n/fcitx5-qt/autobuild/defines (hunk 3 above)
Comment (eatradish) [reply extra-i18n/fcitx5-qt/autobuild/defines]: ok

**Comment by liushuyu** [comment]
Okay, you need to test this PR out before I could merge.

**Review by liushuyu — APPROVED** [review]

**Comment by liushuyu** [comment]
@RedL0tus Anything to add?

**Review by RedL0tus — APPROVED** [review]

**Pull request merged**